drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -364,7 +363,7 @@ static void gps_thread(int dev_ptr)
 				 * application, GPS priority is requested.
 				 */
 				if (drv_data->current_cfg.priority) {
-					k_delayed_work_submit(
+					k_work_schedule(
 						&drv_data->blocked_work,
 						K_SECONDS(GPS_BLOCKED_TIMEOUT));
```

Comment:
Just checking - this is not meant to be restarted if it happens multiple times right? This patch changes the behavior, as the old behavior was to restart the timer. I don't know enough about this module to determine whether that's something that actually can happen

Reply:
AFAICT this work cannot be scheduled again before it is cancelled (or executed) first. The `operation_blocked` flag assures this.